src/checkpt1/checkpt1.css

```diff
@@ -0,0 +1,152 @@
+body{
+  background: url('./img/bg.jpg') no-repeat center center/cover;
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+  box-sizing: border-box;
+  height: 100%;
+}
+.container{
```

Comment:
being picky but lets add a space between the className and curly bracket
`.container {`